### PR TITLE
Cosmetic stuff

### PR DESCRIPTION
**Pull request by Kmc2000**
### Intent of your Pull Request
Adds in new cosmetics!

List:
Smart watches, which are PDAs for your wrists
A new jean type
A camo hoodie
Some new, admin only trainers
~~A mobile phone version of the PDA~~ removed for lore concerns

Other stuff:
You can now CTRL+click a PDA to open its menu (providing you're close enough to it)
  This means you can use a PDA without needing to take it out, so if you use the watch or pipboy, you can open it up like this. You CAN also do this with a mousedrag, but CTRL is more convenient
~~Renames the DME so that you don't get that annoying "SAFETY CHECK" thing popup with endless YES or NO boxes (this may just be me and my byond settings, so if you want me to revert that i'm more than happy to)~~
Added some new clothes to the vendomat, including smartwatches

The HOP office can reskin your PDA into a phone or smartwatch if you prefer them, two smartwatches also start in the clothesmate now
   
 NOTE:
The exempt var is just to make phones/watches/gimmick PDAs exempt from going from:

Smartphone -> PDA-meme (assistant)

Pics or it didn't happen:

Custom notification icons:

https://gyazo.com/1f7854f26ec046aa3c4d20961cbcb297

https://gyazo.com/7ff027ea4f68051a97b54c722bf0024b

#### Changelog

:cl:
rscadd: Added in smartwatches! check the hop's PDA painter or a clothesmate to find them
rscadd: Misc clothing items, check the clothesmate!
tweak: Changed a grammar error with the emergency hotline PDA feature
/:cl:

### CODE REVIEW

**Comment by matskuman5** [comment]
Don't really like this

Lore-wise it's 2567, not 2017

**Comment by Kmc2000** [comment]
~~Mats, we are not a MEDrp server...~~

Changed boop

**Comment by Jc1469** [comment]
admin only cruise ships?

and ftr im against all of this. I was against the trainers really, but this is a bit much.

>Lore-wise it's 2567, not 2017

**Comment by fonedsky** [comment]
Who said mobile phone or smartwatches aren't a 2567 design? 
Sure, their design will almost certainly change, but man, PDAs look so big and hefty compared to what you'd ideally see in the very, very far future. I honestly just expected like a small pocket thing or a smartwatch-like device, which is what this is trying to add.
Also, we've had clothes like jeans and pants for centuries. If those designs can last, then so can trainers. Not everything needs to be Super New! and Super Futuristic!
I do agree that the watches or the phones should look a little sleeker or maybe slightly different from what we have today, but you shouldn't eliminate them entirely (seriously, PDAs look ugly, the PDA interface is ugly, those are the sorts of things I expect to see in 2100 or something)

**Comment by Kmc2000** [comment]
agree with fonedsky, but I can see their point as well.

**Comment by fonedsky** [comment]
It doesn't help that certain codebases have laptops and stuff and those are the higher roleplay servers. Efficient designs last a loooooooooooong time, and things from our year can certainly be improved on, but I don't think they'll change a lot.
Also, it's 2567. We don't even know if we'll last that long in real life. We don't even know how things will look like then. It's perfectly fine to take some creative liberties and add in some things we're familiar with in real life to help with a sense of alienation.
This isn't a server focused on roleplaying. Sure, people can do it, and Mats wants to push for med-RP, but not everything, as I said, needs to look advanced and further ahead than we have today.

**Comment by matskuman5** [comment]
It's not just the lore, it's the overall art style of SS13

It's more retro-futuristic than just this clean, sleek futuristic you see everywhere

PDAs could be reskinned, yes. But one sprite not fitting doesn't mean the other sprite is OK

**Comment by Kmc2000** [comment]
@matskuman5

Phones are no longer a part of this,  is this good now?

**Comment by Super3222** [comment]
Cosmetic PR's are my favorite.

Both Mats and Foned are correct in some way.

It doesn't matter if it fits the lore of it being 2567, but at the same time it does. Honestly, if you could make those watches look more future-y then people would get along with them better.

Kmc nails down the sprite style on some of those clothes, but the shoes are always off. You realllyyyy need to make them look isometric. Look at the laceup shoe design if you want to know what I mean ( or any shoe design in general). We shouldn't be throwing art style in a whole new direction. Those shouldn't be merged until that's fixed. We let you off last time

**Comment by Kmc2000** [comment]
sack the shitty shoes xdddddddddd

**Comment by ShadowDeath6** [comment]
2557 >>>>> 2567

**Comment by ShadowDeath6** [comment]
2557>2567

Also I have no idea how you got away with naming the path of the shoes /yeezy, codermans off their review game

**Comment by Kmc2000** [comment]
>Also I have no idea how you got away with naming the path of the shoes /yeezy, codermans off their review game

Paths don't matter no one ever sees them....

**Comment by ShadowDeath6** [comment]
Doesn't mean it doesn't annoy people to see a dumb meme path tho >:

**Comment by Kmc2000** [comment]
Meh

**Comment by Lightning38088** [comment]
will donors get the option to reskin thier PDAs instead of going to the HoP like everyone else?

**Comment by fonedsky** [comment]
Shadow, I don't think anyone gives a crap.
Kmc is a busy guy. He's probably stressed by the wonders of coding AND adminning at the same time. This goes the same for coders. I think it's okay for them to do dumb coder memes occasionally if it won't affect anything.

**Comment by ShadowDeath6** [comment]
being busy isn't an excuse for writing bad code
paths are only seen by coders so it's a meme that gets stale and people get confused because they open shoecode and find a subtype of sneakers that are called yeezies and have no idea what the difference between them is

**Comment by Kmc2000** [comment]
>being busy isn't an excuse for writing bad code and paths are only seen by coders so it's a meme that gets stale and people get confused because they open shoecode and find a subtype of sneakers that are called yeezies and have no idea what the difference between them is

Give me an alternative, and I will repath them.

**Comment by Kmc2000** [comment]
>will donors get the option to reskin thier PDAs instead of going to the HoP like everyone else?

In the future, yes. But configs are dangerous so I may ask Cruix.

**Comment by fonedsky** [comment]
You have a point. I didn't think it'd be detrimental, just that you were shitting on the guy for doing a dumb meme.

**Comment by matskuman5** [comment]
>Kmc is a busy guy. He's probably stressed by the wonders of coding AND adminning at the same time.

> the wonders of coding AND adminning at the same time

> adminning

Ha

**Comment by ShadowDeath6** [comment]
do /trainers or something

**Comment by Super3222** [comment]
I love cosmetics, BUT FIX THE DAMN SHOES! :-1:

**Comment by Kmc2000** [comment]
Reeeee


REEE I removed them :^)

**Comment by Lightning38088** [comment]
Those yeezies reminded me of black/grey clown shoes when people walked left or right.

**Comment by Super3222** [comment]
Smartphones are dumb. :-1: I agree with anyone that doesn't like their look; wouldn't fit into the game at all. Especially when it's a reskin option for your PDA.
Watches are eh, given that we have lots of classy non-space themed designs already installed in the game.
Jeans and hoodies look sweet.

File: code/game/objects/items/devices/PDA/PDA_types.dm [hunk 1]
```diff
@@ -27,6 +27,49 @@
 /obj/item/device/pda/ai/pai
 	ttone = "assist"
 
+/obj/item/device/pda/iwatch
```
Comment (Super3222) [inline code/game/objects/items/devices/PDA/PDA_types.dm:30]: Can you make the watch like. a device, and not a PDA. something that people can look at to tell the time.

File: code/game/objects/items/devices/PDA/PDA_types.dm [hunk 2]
```diff
@@ -27,6 +27,49 @@
...
+
+/obj/item/device/pda/iwatch/rosegold/examine(mob/user) //tells the time, yes it's a freaking watch duhhh
+	user << desc
+	user << " The time is: [gameTimestamp(format = "hh:mm:ss")]"
+
+//ACKNOWLEDGEMENT: The buzz sound for the phone and watches can be found here: https://www.youtube.com/watch?v=zn8ORi6UUJE
+
+
+
+//obj/item/device/pda/phone
```
Comment (Super3222) [inline code/game/objects/items/devices/PDA/PDA_types.dm:61]: delete this please

File: code/game/objects/items/devices/PDA/PDA_types.dm [hunk 3]
```diff
@@ -27,6 +27,49 @@
...
+	slot_flags = SLOT_GLOVES
+	desc = "A watch with inbuilt computer functions, it has a pink tinge and is perfect for the ladies, ctrl click it to use it whilst on your wrists."
+	exempt = 1 //exempt, will not update name the same way as normal pdas when ID added
+	icon_alert = "pda-rwatch" //custom alert icon for a watch that's not as gross
+
+/obj/item/device/pda/iwatch/rosegold/examine(mob/user) //tells the time, yes it's a freaking watch duhhh
+	user << desc
+	user << " The time is: [gameTimestamp(format = "hh:mm:ss")]"
+
+//ACKNOWLEDGEMENT: The buzz sound for the phone and watches can be found here: https://www.youtube.com/watch?v=zn8ORi6UUJE
```
Comment (Super3222) [inline code/game/objects/items/devices/PDA/PDA_types.dm:57]: it was a free phone vibrate noise. do not reference it

**Pull request merged**